TestClasses/ReadWriteTest.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import sys
+import yaml
+from BaseTest import BaseTest
+from PerformanceTest import PerformanceTest
+import subprocess
+import os
+import time
+import numpy as np
+from zlib import adler32
+
+class ReadWriteTest(BaseTest, PerformanceTest):
+    def __init__(self, configFile=None, singleEndPoint=None, singleProt:list=None):
+        super().__init__()
+        self.groupedOutput = []
+        self.checksums = []
+
+        if configFile is not None:
+            with open(configFile) as config:
+                try:
+                    sites = yaml.safe_load(config)
+                    self.sitesList = sites['SITES']
+                except yaml.YAMLError as exc:
+                    print(exc)
+        
+        if singleEndPoint is not None:
+            self.sitesList = singleEndPoint
+        if singleProt is not None:
+            self.protocols = singleProt
+        #sys.exit()
+
+    
+    def subprocess(self, action:str, sourcePath=None, destinBaseNm=None, *args, timeout=None):
```

Comment:
subprocess is a default python function. unless the function name needs to be that a different name might be better